utils/github/parser.py

```diff
@@ -10,6 +10,10 @@ class Description:
         'Performance Improvement': 'pr-performance',
         # 'Backward Incompatible Change': doesn't match anything
         'Build/Testing/Packaging Improvement': 'pr-build',
+        'Non-significant (changelog entry is not needed)': 'pr-non-significant',
```

Comment:
Maybe reuse `pr-other` tag? It's shorter.

Reply:
I think that tag `pr-non-significant` is less surprising than `pr-other` while converting from this description.